ibctest/relay_many_test.go

```diff
@@ -0,0 +1,197 @@
+package ibctest_test
+
+import (
+	"context"
+	"testing"
+
+	transfertypes "github.com/cosmos/ibc-go/v5/modules/apps/transfer/types"
+	relayeribctest "github.com/cosmos/relayer/v2/ibctest"
+	ibctest "github.com/strangelove-ventures/ibctest/v5"
+	"github.com/strangelove-ventures/ibctest/v5/chain/cosmos"
+	"github.com/strangelove-ventures/ibctest/v5/ibc"
+	ibctestrelayer "github.com/strangelove-ventures/ibctest/v5/relayer"
+	"github.com/strangelove-ventures/ibctest/v5/test"
+	"github.com/strangelove-ventures/ibctest/v5/testreporter"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap/zaptest"
+	"golang.org/x/sync/errgroup"
+)
+
+// TestRelayerMultiplePathsSingleProcess tests relaying multiple paths
+// from the same process using the go relayer. A single
+// CosmosChainProcessor (gaia) will feed data to two PathProcessors (gaia-osmosis and gaia-juno).
+func TestRelayerMultiplePathsSingleProcess(t *testing.T) {
+	relayeribctest.BuildRelayerImage(t)
+
+	client, network := ibctest.DockerSetup(t)
+	r := ibctest.NewBuiltinRelayerFactory(
+		ibc.CosmosRly,
+		zaptest.NewLogger(t),
+		ibctestrelayer.CustomDockerImage(relayeribctest.RelayerImageName, "latest", "100:1000"),
+		ibctestrelayer.ImagePull(false),
+		ibctestrelayer.StartupFlags("-p", "events"),
```

Comment:
This line could be dropped after #989 is merged

Reply:
will include this in #989